openpype/hosts/maya/plugins/publish/extract_look.py

```diff
@@ -176,6 +177,24 @@ def process(self,
             source
         ]
 
+        # if color management is enabled we pass color space information
+        if color_management["enabled"]:
+            config_path = color_management["config"]
+            if not os.path.exists(config_path):
+                raise RuntimeError("OCIO config not found at: "
+                                   "{}".format(config_path))
+
+            if not os.getenv("OCIO"):
+                self.log.warning(
+                    "OCIO environment variable not set."
+                    "Setting it with OCIO config from OpenPype/AYON Settings."
+                )
```

Comment:
I believe this warning is incorrect - you'd actually be setting it to currently configured OCIO path that Maya is set to - not specifically what project settings is set to. The color management being passed along I believe comes from [here](https://github.com/ynput/OpenPype/blob/499b4ddae529f2d2f2a880df08e7ff0bb320f617/openpype/hosts/maya/plugins/publish/extract_look.py#L615) which [queries it from maya](https://github.com/ynput/OpenPype/blob/499b4ddae529f2d2f2a880df08e7ff0bb320f617/openpype/hosts/maya/api/lib.py#L3779-L3831) and thus technically avoids OpenPype settigns completely.